DigitalLearningSolutions.Data/DataServices/LearningResourceReferenceDataService.cs

```diff
@@ -35,6 +37,16 @@ FROM LearningResourceReferences
             ).Single();
         }
 
+        public string? GetLearningHubResourceLinkByResourceRefId(int learningResourceReferenceId)

```

Comment:
I don't think we'll need this if we're using LearningHubResourceService instead.